build/Linux+BSD/portable/copy-libs

```diff
@@ -93,6 +93,8 @@ getCrossPlatformDependencies() {
 
   copyQt plugins/sqldrivers/libqsqlite.so
 
+  copyQt plugins/xcbglintegrations/libqxcb-glx-integration.so
```

Comment:
Thanks for finding this!

You added it to [getCrossPlatformDependencies()](https://github.com/ericfont/MuseScore/blob/101171-AppImage-plugin-crash/build/Linux%2BBSD/portable/copy-libs#L40) function, but I can't see an equivalent in [mscore/CMakeLists.txt](https://github.com/musescore/MuseScore/blob/767a95a9db9cb0b95586af69fbc07c88bb33813b/mscore/CMakeLists.txt#L403-L474). If it doesn't have an equivalent on Windows then it belongs in the [building_on_CentOS_6()](https://github.com/ericfont/MuseScore/blob/101171-AppImage-plugin-crash/build/Linux%2BBSD/portable/copy-libs#L112) function (and it goes in the "#Needed by Centos6" section if it is not present on a clean CentOS 6).

Maybe add a comment so we know why it is needed:
<pre>
copyQt plugins/xcbglintegrations/libqxcb-glx-integration.so # Needed by plugin "helloqml"
</pre>

If there is a Windows equivalent (sorry I didn't see it!) then you don't need a comment unless the Windows version is named something significantly different, like [this](https://github.com/musescore/MuseScore/blob/master/build/Linux%2BBSD/portable/copy-libs#L92).